src/controllers/company.number.controller.ts

```diff
@@ -22,21 +22,27 @@ const preValidators = [
 const padCompanyNumber = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
   let companyNumber: string = req.body.companyNumber;
   if (/^([a-zA-Z]{2}?)/gm.test(companyNumber)) {
-    const leadingLetters = companyNumber.substring(0, 2);
-    let trailingChars = companyNumber.substring(2, companyNumber.length);
-    trailingChars = trailingChars.padStart(6, "0");
-    companyNumber = leadingLetters + trailingChars;
+    companyNumber = formatCompanyNumber(companyNumber, 2, 6);
```

Comment:
If we know a company number is always 8 chars long (I think it is, and the two scenarios here suggest that), could you dispense with the second parameter to your 'formatCompanyNumber' function and just calculate the 'padStart' variable as '8 - leadPoint'?

Reply:
Updated.